### ISSUE TITLE
[BUG] Fluxnode 3.28 wont start due wrong Syncthing config

### ISSUE THREAD

**Issue by HadiJadallah** [RunOnFlux/flux#855]
**Describe the bug**
On fresh install of 3.28 (latest) on update Ubuntu 20.04 Desktop, the following is observed:
 - Log files are filled with Trying to connect to 127.0.0.1:8384 with Connection refused errors
 - Flux report Syncthing is not running properly and will initiate reinstallation of Syncthing 
 - Benchmark will fail and no hardware specs are detected
 - Fluxnode will not sync properly and cannot be started
 - errors loop continuously 

**To Reproduce**
1- Format and install fresh copy Ubuntu Desktop 20.04
2- Follow guide https://medium.com/@mmalik4/flux-light-node-setup-as-easy-as-it-gets-833f17c73dbb to setup
3- tail -f the log files or user flux_node_viewer.sh -> press L -> observe

**Expected behavior**
Syncthing should work on default config

**Environment(please complete the following information):**
 - OS: Ubuntu 20.04 LTS Desktop
 - Browser
 - Version 
 - Daemon 6000050
 - Benchmark 3.6

**Additional context**
Possible Fixes:
1- 
Changing the ~/.config/syncthing/config.xml GUI Listen port from 41955 to 8384 as follows:
NOTE: I changed config.xml.v0 as well

`<gui enabled="true" tls="false" debugging="false">`
`<address>127.0.0.1:41955</address>`
`</gui>`
to 
`<gui enabled="true" tls="false" debugging="false">`
`<address>127.0.0.1:8384</address>`
`</gui>`

Fixes the issue, Syncthing runs and benchmark completes correctly.

Per Syncthing documentation, the default example configuration references port 8384 which was not the case here.
Also, changing the port from 8384 to port used by Syncthing xml in zelflux/Zelback/config/default.js file will cause issues like version being not supported even though it fixes Syncthing connection issues.

2- Change default syncthing launch arguments to include --gui-address=127.0.0.1:8384
which should override GUI listen address to whatever is specified in zelflux/Zelback/config/default.js

3- Delete  ~/.config/syncthing/ and reinstall also fixes the problem forcing Syncthing to regenerate the correct config

Thanks

**Comment by HadiJadallah** [comment]
You are correct. Another user on discord had the same issue while installing.
In my case am pretty sure I borked the syncthing install somehow.
My journey was:
1- Install ubuntu 20.04 creating user X
2- login user x and initiate flux install
3- Flux install ask for username and switches to new user y
4- I switched back to user y and installed
5- Realized my mistake when flux was not running giving all errors
6- log in user y and delete files in user x (did not realize .config folder exits) and did not purge syncthing
7- install from user y account
Here, my guess is, Syncthing while installing realized another copy is installed already and is listening on 8384 and decided to generate new config with different port.

In all cases, teaching the installer about pre-installed / pre-exiting config syncthing would resolve this issue.
